spec/features/users/merchant_dash_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
```

Comment:
The merge should should include the View Items link referenced in user story (#12).